From c455f80d2473be0ddce51e3e3d6b15401b723e60 Mon Sep 17 00:00:00 2001
From: Ranjani Vaidyanathan <Ranjani.Vaidyanathan@nxp.com>
Date: Mon, 7 May 2018 16:23:59 -0500
Subject: [PATCH 3716/5242] MLK-18220-1 XRDC: Add GPIO clocks to device tree
 node.

commit  c1843f2a931e9c90c416eccce77dac3fcc143c29 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add clocks required by GPIO to the device tree entries.

Signed-off-by: Ranjani Vaidyanathan <Ranjani.Vaidyanathan@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-device.dtsi      |   14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
index fe052b0..22b49fb 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
@@ -2633,6 +2633,8 @@
                 power-domains = <&pd_lsio_gpio0>;
 		interrupt-controller;
 		#interrupt-cells = <2>;
+		clocks = <&clk IMX8QM_GPIO0_IPG_S_CLK>;
+		clock-names = "ipg";
 	};
 
 	gpio1: gpio@5d090000 {
@@ -2644,6 +2646,8 @@
                 power-domains = <&pd_lsio_gpio1>;
 		interrupt-controller;
 		#interrupt-cells = <2>;
+		clocks = <&clk IMX8QM_GPIO1_IPG_S_CLK>;
+		clock-names = "ipg";
 	};
 
 	gpio2: gpio@5d0a0000 {
@@ -2655,6 +2659,8 @@
                 power-domains = <&pd_lsio_gpio2>;
 		interrupt-controller;
 		#interrupt-cells = <2>;
+		clocks = <&clk IMX8QM_GPIO2_IPG_S_CLK>;
+		clock-names = "ipg";
 	};
 
 	gpio3: gpio@5d0b0000 {
@@ -2666,6 +2672,8 @@
                 power-domains = <&pd_lsio_gpio3>;
 		interrupt-controller;
 		#interrupt-cells = <2>;
+		clocks = <&clk IMX8QM_GPIO3_IPG_S_CLK>;
+		clock-names = "ipg";
 	};
 
 	gpio4: gpio@5d0c0000 {
@@ -2677,6 +2685,8 @@
                 power-domains = <&pd_lsio_gpio4>;
 		interrupt-controller;
 		#interrupt-cells = <2>;
+		clocks = <&clk IMX8QM_GPIO4_IPG_S_CLK>;
+		clock-names = "ipg";
 	};
 
 	gpio5: gpio@5d0d0000 {
@@ -2699,6 +2709,8 @@
                 power-domains = <&pd_lsio_gpio6>;
 		interrupt-controller;
 		#interrupt-cells = <2>;
+		clocks = <&clk IMX8QM_GPIO6_IPG_S_CLK>;
+		clock-names = "ipg";
 	};
 
 	gpio7: gpio@5d0f0000 {
@@ -2710,6 +2722,8 @@
                 power-domains = <&pd_lsio_gpio7>;
 		interrupt-controller;
 		#interrupt-cells = <2>;
+		clocks = <&clk IMX8QM_GPIO7_IPG_S_CLK>;
+		clock-names = "ipg";
 	};
 
 	gpio0_mipi_csi0: gpio@58222000 {
-- 
1.7.9.5

